example/09_convnd_fwd/convnd_fwd_common.hpp

```diff
@@ -269,6 +274,235 @@ bool run_grouped_conv_fwd(bool do_verification,
                                     get_rtol<OutDataType, ComputeDataType>(),
                                     get_atol<OutDataType, ComputeDataType>());
     }
+    else if(do_verification == 2)
+    {
+        // GPU verification using naive GPU reference
+        std::cout << "Running GPU verification..." << std::endl;
+
+        // Allocate and ZERO GPU memory for reference output
+        DeviceMem out_device_ref_buf(sizeof(OutDataType) * out_device.mDesc.GetElementSpaceSize());
+        out_device_ref_buf.SetZero();
+
+        // Extract dimensions for old CK GPU reference (NDHWC layout)
+        ck::index_t N_val = conv_param.N_;
```

Comment:
Can we remove these lists and just pass these params to the function? I think that they are self descriptive